release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -2733,6 +2733,40 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.15.25
+[id="ocp-4-15-25_{context}"]
+=== RHSA-2024:4955 - {product-title} 4.15.25 bug fix and security update
+
+Issued: 2024-08-07
+
+{product-title} release 4.15.25, which includes security updates, is now available. The list of bug fixes that are included in this update is documented in the link:https://access.redhat.com/errata/RHSA-2024:4955[RHSA-2024:4955] advisory. The RPM packages that are included in this update are provided by the link:https://access.redhat.com/errata/RHSA-2024:4958[RHSA-2024:4958] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.15.25 --pullspecs
+----
+
+[id="ocp-4-15-25-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, a machine config pool (MCP) with a higher `maxUnavailable` value than the amount of unavailable nodes would cause cordoned nodes to receive updates if they were in a certain position on the node list. With this release, a fix ensures that cordoned nodes are not added to a queue to receive updates. (link:https://issues.redhat.com/browse/OCPBUGS-37629[*OCPBUGS-37629*])
+
+* Prevously, an errant code change resulted in a duplicated `oauth.config.openshift.io` item on the Global Configuration page. With this release, the duplicated item is removed. (link:https://issues.redhat.com/browse/OCPBUGS-37458[*OCPBUGS-37458*])
+
+* Previously, the Cluster Network Operator (CNO) IPsec mechanism incorrectly behaved on a cluster that had multiple worker machine config pools. With this release, CNO Ipsec mechanism works as intended for a cluster with multiple worker machine config pools. This fix does not apply to upgrading IPsec-enabled cluster with multiple paused machine config pools. (link:https://issues.redhat.com/browse/OCPBUGS-37205[*OCPBUGS-37205*])
```

Comment:
Update is the preferred term in our docs:
```suggestion
* Previously, the Cluster Network Operator (CNO) IPsec mechanism incorrectly behaved on a cluster that had multiple worker machine config pools. With this release, CNO Ipsec mechanism works as intended for a cluster with multiple worker machine config pools. This fix does not apply to updating IPsec-enabled cluster with multiple paused machine config pools. (link:https://issues.redhat.com/browse/OCPBUGS-37205[*OCPBUGS-37205*])
```

Reply:
:heavy_check_mark: